frontend/wikipedia_home.py

```diff
@@ -23,3 +23,12 @@ def anyone_can_edit(self):
         for tag in all_a_tags:
             if tag.get_attribute("title") == "Help:Introduction to Wikipedia":
                 return tag
+
+    @property
+    def search_bar(self):
+        return self.driver.find_element(By.CLASS_NAME, "cdx-text-input__input")
```

Comment:
Please, use By.ID here. If an element has an ID, it is the best way to search it. 

Here is the priority: 
ID -> NAME -> CLASS NAME -> TAG NAME -> LINK TEXT -> CSS_SELECTOR -> XPATH 
You can read more here: https://www.qafox.com/selenium-locators-priority/